File: Java/src/exercicios/java/atleta/Atleta.java
package exercicios.java.atleta;

public class Atleta {
    private int idade;
    private String nacionalidade;
    private String nome;


    public Atleta(int idade, String nacionalidade, String nome){
        this.idade = idade;
        this.nacionalidade = nacionalidade;
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public String getNacionalidade() {
        return nacionalidade;
    }

    public void setNacionalidade(String nacionalidade) {
        this.nacionalidade = nacionalidade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }


}

File: Python/Atleta.py
class Atleta:
  def __init__(self, idade, nome):
    self.__idade = idade
    self.__nome = nome

  def getIdade(self):
    return self.__idade
  
  def setIdade(self, idade):
    self.__idade = idade

  def getNome(self):
    return self.__nome
    
  def setNome(self, nome):
    self.__nome = nome